README.md

```diff
@@ -15,36 +15,37 @@ The analytics interfaces has to be configured in Webpack configuration file thro
 
 ```javascript
 new webpack.DefinePlugin({
-  __analytics: JSON.stringify({
+  __analytics: {
     debug: {
       isEnabled: true
     },
     googleAnalytics: {
       isEnabled: true,
-      key: '<your-key>'
+      key: JSON.stringify('<your-key>')
     },
     googleAdWords: {
       isEnabled: true,
-      key: '<your-key>'
+      key: JSON.stringify('<your-key>')
     },
     googleTagManager: {
       isEnabled: true
     },
     amplitude: {
       isEnabled: true,
-      key: '<your-key>',
-      allowProprties: [], // optional
-      disallowProperties: ['application', 'user', 'section', 'event_details'], // optional
+      key: JSON.stringify('<your-key>'),
```

Comment:
is there a need to `stringify` key, which is a string originally? 🤔  :D

Reply:
for webpack isn' string ;) https://webpack.js.org/plugins/define-plugin/
![Zrzut ekranu 2022-02-1 o 13 55 47](https://user-images.githubusercontent.com/12138170/151972259-85da5eae-f29d-451d-a666-be25f3c147cc.png)

Reply:
@pspaczek @AlanLes 
WDYT about change only .env variables without any JSON.stringify? based on definePlugin docs we can define variable as below: 
```
ANALYTICS_INFERMEDICA_TOPIC='"YOUR_TOPIC"'
```

Note that because the plugin does a direct text replacement, the value given to it must include actual quotes inside of the string itself. Typically, this is done either with alternate quotes, such as '"production"', or by using JSON.stringify('production').

But i'm not sure about impact on infra, probably they will need to update all variables?